project.clj

```diff
@@ -7,30 +7,30 @@
                         ["snapshots" :clojars]]
   :plugins [[lein-ancient "0.7.0"]
             [lein-nvd "1.4.1"]]
-  :dependencies [[org.clojure/clojure "1.10.3"]
-                 [org.clojure/core.async "1.3.610" :exclusions [org.clojure/tools.reader]]
-                 [prismatic/schema "1.1.12"]
+  :dependencies [[org.clojure/clojure "1.11.1"]
+                 [org.clojure/core.async "1.5.648" :exclusions [org.clojure/tools.reader]]
+                 [prismatic/schema "1.2.0"]
                  [camel-snake-kebab "0.4.2"]
-                 [com.cognitect/transit-clj "1.0.324"]
-                 [com.cognitect/transit-cljs "0.8.264"]
+                 [com.cognitect/transit-clj "1.0.329"]
+                 [com.cognitect/transit-cljs "0.8.269"]
                  [commons-codec/commons-codec "1.15"]
-                 [metosin/ring-http-response "0.9.2" :exclusions [ring/ring-core]]
-                 [org.apache.commons/commons-csv "1.8"]
-                 [commons-io/commons-io "2.8.0"]
-                 [info.sunng/ring-jetty9-adapter "0.14.3" :exclusions [org.eclipse.jetty/jetty-alpn-conscrypt-server
+                 [metosin/ring-http-response "0.9.3" :exclusions [ring/ring-core]]
+                 [org.apache.commons/commons-csv "1.9.0"]
+                 [commons-io/commons-io "2.11.0"]
+                 [info.sunng/ring-jetty9-adapter "0.17.6" :exclusions [org.eclipse.jetty/jetty-alpn-conscrypt-server
```

Comment:
This version does not use `jetty-alpn-conscrypt-server` and `conscrypt-openjdk-uber`.